lib/suse/connect/connection.rb

```diff
@@ -66,6 +66,9 @@ def json_request(method, path, params = {})
           body: body,
           success: response.is_a?(Net::HTTPSuccess)
         )
+
+      rescue Zlib::Error
+        raise SUSE::Connect::NetworkError, 'Network error. Try again and report a bug if it keeps happening.'
```

Comment:
1) `Network error.` is not very useful tbh. Something like `Check your network connection` would be better imho.
2) `and` in second sentence confuses me. It invites me to report bug before I read the condition. `Or` or re-ordering words would be better imho.